android/java/org/chromium/chrome/browser/crypto_wallet/util/TokenUtils.java

```diff
@@ -60,6 +87,9 @@ private static BlockchainToken[] filterTokens(NetworkInfo selectedNetwork,
                 case SOL:
                     typeFilter = t.coin != CoinType.SOL;
                     break;
+                case NON_NFTS:
+                    typeFilter = t.isErc721 || t.isNft;
+                    break;
```

Comment:
Info: Now we also have [`isErc1155` standard](https://bravesoftware.slack.com/archives/C04DTJT4MDG/p1679918029714849?thread_ts=1679914338.370169&cid=C04DTJT4MDG) but it seems to be a mix of fungible and non-fungible tokens so might wanna update accordingly if it is fully supported.

Reply:
I'm going to investigate this before merging. Checking also how it's handled on the iOS side.

Reply:
https://github.com/brave/brave-ios/pull/7174/files#diff-d26d442a8ddfd02ace92df9f6e6d52d1e30e51d817c616ff46f8e6e50179e9fa

iOS is also handling NFT filter logic the same as we do, but I'm going to keep an eye on this.
I'm merging now.